### ISSUE TITLE
Unable to find wordpress plugin

### ISSUE THREAD

**Issue by rochana-atapattu** [vuestorefront-community/woocommerce#17]
### Provide a description of requested docs changes

Unable to find the plugin mentioned in

Wordpress config
Install the following open source Wordpress Plugin into your WP store:

Please add the plugin name

### Able to fix / change the documentation?

- [ ] Yes
- [X] No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chrislemmer** [comment]
Hey Rochana,

Thanks for your response.

We made the plugin private for now. It's still a work in progress.

Once our BETA is ready, we'll share some more code with the community.

Are you interested in assisting with development? Or were you just browsing around?

Cheers
Chris

**Comment by rochana-atapattu** [comment]
Hi Chris

Thank you for the update. 
Yes I am interested in contributing the project.

**Comment by jack-of-blades** [comment]
> Is the plugin still required after this? https://developer.woocommerce.com/2022/03/25/store-api-is-now-considered-stable/

Yes the plugin will still be required as we're building the API specifically for VSF. As part of our plan to keep the VSF integration as performative and as agnostic as possible in the front-end, we're doing the translation in the plugin.

Thanks for your interest!

**Comment by no-1ne** [comment]
Sorry for the ping again, it appears  all the additional API's being added via https://github.com/SwiftComZA/vsf-wc-api   can be pulled off with offical API's end points (without having to add any additional plugin)..even the cart API's which seem to be missing are added as well  https://developer.woocommerce.com/2022/03/25/store-api-is-now-considered-stable/

Create customer, https://woocommerce.github.io/woocommerce-rest-api-docs/?javascript#create-a-customer, may be there is something i am missing can you pls share what particualar API needs to be added
